models/repo.go

```diff
@@ -1666,6 +1693,125 @@ func GitGcRepos() error {
 		})
 }
 
+// Maybe move this to gogits/git-module
```

Comment:
I just copy/paste the comments because in the same file there is also another block to move.
Without comments and depending on the time for the PR to get merge, I will probably forget this (because I haven't only one PR)